### ISSUE TITLE
Investigate node-kubelet-serial-alpha tab - why test not showing up and whether we need this tab

### ISSUE THREAD

**Issue by SergeyKanzhelev** [kubernetes/test-infra#18972]
**What should be cleaned up or changed**:
The tab [node-kubelet-serial-alpha](https://testgrid.k8s.io/sig-node-kubelet#node-kubelet-serial-alpha) is empty - doesn't run any tests. 

CPUManager tests seems to be matching the definition - has `[Serial]` and `[NodeAlphaFeature:]` [tags](https://github.com/kubernetes/test-infra/blob/bd3b81b6c985971cf10665af396962f1c3136785/config/jobs/kubernetes/sig-node/node-kubelet.yaml#L287):

https://github.com/kubernetes/kubernetes/blob/5708511499fe500ae3b4bbd40204cef382f652e8/test/e2e_node/cpu_manager_test.go#L635-L636

But doesn't show up in the tab and test does not set up the feature flag to run it.

So either need to understand why tests wouldn't run or remove the tab altogether.

**Comment by SergeyKanzhelev** [comment]
Like @alejandrox1 said this test indeed didn't run because:

```
I0824 16:58:49.178] Aug 24 16:58:44.299: INFO: Skipping CPU Manager tests since the CPU capacity < 2
```

**Comment by SergeyKanzhelev** [comment]
So CPU manager tests already configured to run in dedicated tab [kubelet-serial-gce-e2e-cpu-manager](https://testgrid.k8s.io/sig-node-kubelet#kubelet-serial-gce-e2e-cpu-manager). So this tab has no tests to run. I'd suggest to delete this tab altogether.

**Comment by SergeyKanzhelev** [comment]
/close

it was resolved with the #18974
